aspnetcore/toc.md

```diff
@@ -1,223 +1,201 @@
-#### [Add search](xref:tutorials/first-mvc-app-mac/search)
-#### [Add a new field](xref:tutorials/first-mvc-app-mac/new-field)
-#### [Add validation](xref:tutorials/first-mvc-app-mac/validation)
-#### [Examine the Details and Delete methods](xref:tutorials/first-mvc-app/details)
-
-### [MVC web app with Visual Studio Code on macOS or Linux](xref:tutorials/first-mvc-app-xplat/index)
-#### [Get started](tutorials/first-mvc-app-xplat/start-mvc.md)
-#### [Add a controller](tutorials/first-mvc-app-xplat/adding-controller.md)
-#### [Add a view](tutorials/first-mvc-app-xplat/adding-view.md)
-#### [Add a model](tutorials/first-mvc-app-xplat/adding-model.md)
-#### [Work with SQLite](tutorials/first-mvc-app-xplat/working-with-sql.md)
-#### [Controller methods and views](tutorials/first-mvc-app-xplat/controller-methods-views.md)
-#### [Add search](tutorials/first-mvc-app-xplat/search.md)
-#### [Add a new field](tutorials/first-mvc-app-xplat/new-field.md)
-#### [Add validation](tutorials/first-mvc-app-xplat/validation.md)
-#### [Examine the Details and Delete methods](tutorials/first-mvc-app/details.md)
-
-### [Web API with Visual Studio for Mac](xref:tutorials/first-web-api-mac)
-### [Web API with Visual Studio Code](xref:tutorials/web-api-vsc)
-
-## [Develop apps using a file watcher](xref:tutorials/dotnet-watch)
-
-## [Create backend services for mobile apps](mobile/native-mobile-backend.md)
-
-# [Fundamentals](xref:fundamentals/index)
-## [Application startup](xref:fundamentals/startup)
+# Fundamentals
+## [Overview](xref:fundamentals/index)
+## [App startup](xref:fundamentals/startup)
 ## [Dependency injection (services)](xref:fundamentals/dependency-injection)
-## [Middleware](xref:fundamentals/middleware/index)
-### [Middleware](xref:fundamentals/middleware/index)
-### [Factory-based middleware](xref:fundamentals/middleware/extensibility)
-### [Factory-based middleware with third-party container](xref:fundamentals/middleware/extensibility-third-party-container)
-## [Static files](xref:fundamentals/static-files)
 ## [Routing](xref:fundamentals/routing)
-## [URL rewriting](xref:fundamentals/url-rewriting)
-## [Use multiple environments](xref:fundamentals/environments)
-## [Configuration and options](xref:fundamentals/configuration/index)
-### [Configuration](xref:fundamentals/configuration/index)
-### [Options](xref:fundamentals/configuration/options)
+## [Multiple environments](xref:fundamentals/environments)
+## [Configuration](xref:fundamentals/configuration/index)
+## [Options](xref:fundamentals/configuration/options)
 ## [Logging](xref:fundamentals/logging/index)
-### [Logging with LoggerMessage](xref:fundamentals/logging/loggermessage)
-## [Handle errors](fundamentals/error-handling.md)
-## [Host](xref:fundamentals/host/index)
+## [Handle errors](xref:fundamentals/error-handling)
+## Middleware
+### [Overview](xref:fundamentals/middleware/index)
+### [Factory-based middleware](xref:fundamentals/middleware/extensibility)
```

Comment:
Can the word "middleware" be removed from this link and the one below?

Reply:
@danroth27 We decided that "Middleware" belongs in Fundamentals, but what about the two "factory-based" articles - can those go to Other topics?